### ISSUE TITLE
Bilateral high-frequency sensorineural hearing impairment

### ISSUE THREAD

**Issue by slaulederkind** [obophenotype/human-phenotype-ontology#3572]
_For new term requests, please provide us with the following information:_
### 1. Preferred term label

Bilateral high-frequency sensorineural hearing impairment

### 2. Synonyms

Bilateral high-frequency nerve deafness ;   Bilateral high-frequency sensorineural deafness ;   Bilateral high-frequency sensorineural hearing loss 

### 3. Textual definition (should be understandable even for non-specialists, please include a PubMed ID for relevant articles providing additional information about the suggestion)

A bilateral, high-frequency form of sensorineural hearing impairment.

### 4. Parent term (use [HPO Browser](http://compbio.charite.de/hpoweb/showterm?id=HP:0000118) or [OLS](http://www.ebi.ac.uk/ols/ontologies/hp))

Bilateral sensorineural hearing impairment (HP:0008619)

### 6. Your nano-attribution (ORCID-id or label, e.g. _HPO:probinson_ (organization:name))

RGD:gthayman

**Comment by slaulederkind** [comment]
PMID:9843210

**Comment by pnrobinson** [comment]
Please note that the HPO now has Clinical modifer terms
These can be mixed and matched with existing HPO terms and will avoid combinatorial explosions
Here is the term for Bilateral
https://hpo.jax.org/app/browse/term/HP:0012832
The current HPO has terms that in retrospect should have been made with modifiers (live and learn), but going forward we would like to have explicit pre-composed terms only when the community has an agreed upon meaning (e.g., "Moderate intellectual disability" has its own term because we can provide a definition based on community standards, but "Moderate low-set ears" would not make sense). 
If there is a particular medical subclass based on "Bilateral sensorineural hearing impairment " that would justify a new term, please reopen this issue. Otherwise, please use the Modifier.
